### DISCUSSION TITLE
@codetrix-studio/capacitor-google-auth works with CapV5, this plugin did not

### DISCUSSION THREAD

**Discussion by folsze** [capawesome-team/capacitor-firebase#686] [Q&A] [answered]
Is there any known issues with Capacitor v5? Whenever I called sign in, nothing happened. With the other google library it worked flawlessly, the setup. 

An example project would be nice, since I might want to include more than just google sign in in the future. The other library has lots of example projects online and on medium, it just works. 

Thanks a lot for this library, I really appreciate it, especially once I get it working at some point

**Comment by robingenz** [comment, marked as answer]
It is difficult to help you without having a reproducible example. There can be various reasons why the plugin does not work. Mostly it is because the installation instructions were not followed exactly. Feel free to take a look at our [demo app](https://github.com/robingenz/capacitor-firebase-authentication-demo). Also note that we only support the latest Capacitor major version, currently Capacitor 6.

**Reply by folsze** [reply]
Thanks for the demo. I could not find it. And sorry for this low quality discussion post.

**Reply by folsze** [reply]
Hi,

I found the error. It was what was marked already as important not to miss:

https://github.com/capawesome-team/capacitor-firebase/compare/main...folsze:capacitor-firebase:patch-1

I created a OR where I added two red emojis for future users not missing it, I think it's still easy to miss.


Also, so because you said "we only support the latest Capacitor major version, currently Capacitor 6" I upgraded to v6. But now I am facing a new issue:

https://stackoverflow.com/questions/78813231/capacitor-community-sqlite-typeerror-x-is-not-a-function-upon-startup-after-u

I heard you are offering to be the new maintainer of this plugin, which I find extremely great. Maybe this is a first good look into this plugin? Often errors like this came up when I used it throughout cap v3, v4, v5

**Reply by robingenz** [reply]
I'm glad you found a solution. 

>I heard you are offering to be the new maintainer of this plugin, which I find extremely great. Maybe this is a first good look into this plugin? Often errors like this came up when I used it throughout cap v3, v4, v5

Yes, but unfortunately I don't have the time at the moment.
